OpenRA.Game/Settings.cs

```diff
@@ -288,6 +288,19 @@ Hotkey GetKey(Func<KeySettings, Hotkey>[] keys, int index)
 		}
 	}
 
+	public class IrcSettings
+	{
+		public string[] Hostname = { "irc.openra.net" };
+		public int Port = 6667;
+		public string Nickname = "Newbie";
+		public string Username = "OpenRA";
```

Comment:
I agree with @Phrohdoh.  We are using IRC as the backend for our global chat, not writing a general purpose IRC client.  There's no benefit to making this configurable.